### ISSUE TITLE
crm_resource -? output has changed in pcmk master

### ISSUE THREAD

**Issue by mbaldessari** [ClusterLabs/pcs#260]
I noticed this by pure accident, raising it as it might save some time/investigation.

`crm_resource -?` which is used in pcs/lib/pacemaker/live.py to test for features has been changed in pcmk on master and it does not output all the options any longer, so naturally the following will fail:
~~~
pcs resource disable --wait=600 haproxy-bundle
Error: crm_resource does not support --wait, please upgrade pacemaker
~~~

I quickly fixed it locally by replacing "crm_resource -?" with "crm_resource --help-all" in pcs/lib/pacemaker/live.py  and my commands were working again. (I checked and crm_resource --help-all works in pcmk 1.1 as well, fwiw)

**Comment by tomjelinek** [comment]
@mbaldessari Thank you for the heads-up. However, `man crm_resource` mentions `--help`, so I think we should go with that. `--help-anything` seems to be giving the same output as `--help`.

@kgaillot  Can you confirm this? Is `-?` really being removed? Thanks.

**Comment by tomjelinek** [comment]
@kgaillot Thanks for detailed explanation.

Pcs currently uses `crm_resource -?` to look for `--wait` (Wait until the cluster settles into a stable state) and `--expired` (remove constraints with expired lifetimes). For other tools, it uses `--help-all`: `crm_mon --help-all` to look for `--fence-history` and `stonith-admin --help-all` to look for `--history`, `--broadcast` and `--cleanup`.

Since `--help-all` works with the legacy help system, we can switch `crm_resource -?` to `crm_resource --help-all`. On top of that, we can drop checks for `--wait` support in `crm_resource`, since it is supported in all Pacemaker 2.x releases and pcs-0.10 branch doesn't support Pacemaker 1.x anyway.

**Comment by mirecheck** [comment]
The issue should be fixed in this commit [363c8b0c](https://github.com/ClusterLabs/pcs/commit/363c8b0c6836bfc918ce1698e448cbffb4f0468e).
